skrifa/Cargo.toml

```diff
@@ -8,9 +8,15 @@ repository = "https://github.com/googlefonts/fontations"
 readme = "README.md"
 categories = ["text-processing", "parsing", "graphics"]
 
+[features]
+serde = ["dep:serde"]
+
 [dependencies]
-read-fonts = { version = "0.14.0", path = "../read-fonts" }
+read-fonts = { version = "0.14.0", path = "../read-fonts", features = ["serde"] }
```

Comment:
This pulls serde in unconditionally for all builds. Let’s remove this and add the serde feature to the read-fonts dev-dependency.

Reply:
That's done now.